src/ClockPanel.tsx

```diff
@@ -109,7 +109,7 @@ class UnthemedClockPanel extends PureComponent<Props, State> {
     }
 
     const timeLeft = moment.duration(
-      moment(countdownSettings.endCountdownTime).utcOffset(this.getTZ(timezone).format('Z'), true).diff(now)
+      moment(this.props.replaceVariables(countdownSettings.endCountdownTime)).utcOffset(this.getTZ(timezone).format('Z'), true).diff(now)
```

Comment:
**Linting:** you will need to format it in the following way to make the tests pass:

 ```suggestion
      moment(this.props.replaceVariables(countdownSettings.endCountdownTime))
        .utcOffset(this.getTZ(timezone).format('Z'), true)
        .diff(now)
```